scalafix-tests/unit/src/test/scala/scalafix/tests/config/ScalaVersionSuite.scala

```diff
@@ -0,0 +1,12 @@
+package scalafix.tests.config
+
+import scalafix.internal.config.ScalaVersion
+
+class ScalaVersionSuite extends munit.FunSuite {
+
+  test("parser Scala2 ") {
+    val scala2 = ScalaVersion.from("2")
```

Comment:
it looks like you added support for many more patterns (I am guessing the RC suffix is gracefully dropped for example), it would be nice to document/test them.

Reply:
I agree.